core/commands/destroy_command.rb

```diff
@@ -46,10 +46,13 @@ def show_help
 Or you can destroy all nodes:
   mdbci destroy configuration
 
+Or you can destroy all configurations in directory:
+  mdbci destroy --all vms-directory-path
```

Comment:
User may specify the `MDBCI_VM_PATH` environment variable. In that directory all the VMs will be created. Consider to use it as the default value too.